src/movies.js

```diff
@@ -1,25 +1,96 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+
+function getAllDirectors(moviesArray){
+    const directorsArray = moviesArray.map(element => element.director, 0);
+    return directorsArray
+}
+
+
+function cleanAllDirectors(moviesArray){
+    let newArr = [];
+    for (let i = 0; i<moviesArray.length; i++){
+        if (newArr !== moviesArray[i].director){
+            newArr.push(moviesArray[i])
+        }
+    }
```

Comment:
Nice jooob alose There is another way to do it, like this 🚀 
```suggestion
 return [...new Set(moviesArray)]
    }
```